src/intrinsics.h

```diff
@@ -152,21 +153,25 @@ extern "C"
 JL_CALLABLE(jl_f_intrinsic_call)
 {
     JL_NARGSV(intrinsic_call, 1);
-    JL_TYPECHK(intrinsic_call, intrinsic, args[0]);
-    enum intrinsic f = (enum intrinsic)*(uint32_t*)jl_data_ptr(args[0]);
+    JL_TYPECHK(intrinsic_call, intrinsic, F);
```

Comment:
`JL_CALLABLE`. `args` comes from the same place. I personally agree it's confusing and one way to make it more explicit is to pass in the variable names a macro parameters, i.e. `JL_CALLABLE(..., F, args)`.

Anyway, unrelated to this PR.